### DISCUSSION TITLE
define-your-schemas documentation

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [fastapi-users/fastapi-users#1194] [General]
https://fastapi-users.github.io/fastapi-users/10.4/configuration/schemas/#define-your-schemas

Says, "There are four Pydantic models variations provided as mixins:"

But, only four are described: BaseUser, BaseCreateUser and BaseUpdateUser.

**Comment by frankie567** [comment]
Yes, that's a relicate from an old version, there is only *three* schemas now. PR are welcome to fix this 😄
